packages/components/organisms/f-cookie-banner/package.json

```diff
@@ -41,13 +41,14 @@
   ],
   "dependencies": {
     "@justeat/f-globalisation": "1.0.0",
-    "@justeat/f-services": "1.7.0"
+    "@justeat/f-services": "1.7.0",
+    "js-cookie": "2.2.1"
```

Comment:
Can we set this as a `peerDependency` [like we do in the OIDC package](https://github.com/justeat/fozzie-components/blob/200d53a4bee1eae4a3841cbb0c6e7a400e557336/packages/services/f-consumer-oidc/package.json#L44)? 

You'd also need to mark `js-cookie` as external, [like we do in `f-mega-modal`](https://github.com/justeat/fozzie-components/blob/200d53a4bee1eae4a3841cbb0c6e7a400e557336/packages/components/molecules/f-mega-modal/vue.config.js#L20-L22).

That would mean that the consuming application is responsible for providing the `js-cookie` npm package and we don't have to bundle it in the `f-cookie-banner` package? This helps to reduce duplicated content inside npm packages.

Reply:
Done